src/com/owncloud/android/ui/activity/LocalFolderPickerActivity.java

```diff
@@ -121,6 +123,17 @@ public void onClick(View view) {
                 finish();
             }
         });
+        mHomeBtn = findViewById(R.id.folder_picker_btn_home);
+        mHomeBtn.setVisibility(View.VISIBLE);
```

Comment:
Do you really need to take care of this button visibility?  You are defining it like gone in the xml and making it visible again.

Reply:
Yes but its necesry because the layout is used twice in the code. Once for moving files within the cloud and once to select the folder for the camera upload.

We don't want the button to show up when moving files.